code/modules/mob/living/simple_animal/bot/ed209bot.dm

```diff
@@ -1,7 +1,7 @@
 /mob/living/simple_animal/bot/secbot/ed209
 	name = "\improper ED-209 Security Robot"
 	desc = "A security robot. He looks less than thrilled."
-	icon_state = "ed2090"
+	icon_state = "ed209"
```

Comment:
the extra number at the end gets added by the update_icon proc so it's not needed here